### PR TITLE
For discussion: be explicit about JRuby opts

### PR DESCRIPTION
**Pull request by bf4**
This is really only an issue on JRUBY-HEAD where
cext.enabled property no longer exists.

https://travis-ci.org/rspec/rspec-core/jobs/50010459#L166

### CODE REVIEW

**Comment by myronmarston** [comment]
Thanks.  I used the `travis:create_pr_with_updates` task to create a PR in each repo with these updates.

File: travis/script/functions.sh [hunk 1]
```diff
@@ -2,8 +2,9 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $SCRIPT_DIR/travis_functions.sh
 source $SCRIPT_DIR/predicate_functions.sh
 
-# idea taken from: http://blog.headius.com/2010/03/jruby-startup-time-tips.html
-export JRUBY_OPTS="${JRUBY_OPTS} -X-C" # disable JIT since these processes are so short lived
+# If JRUBY_OPTS isn't set, use these.
+# see http://docs.travis-ci.com/user/ci-environment/
+export JRUBY_OPTS=${JRUBY_OPTS:---server -Xcompile.invokedynamic=false"}
```
Comment (myronmarston) [inline travis/script/functions.sh]: It appears that you're change here is invalid bash, as the PRs I opened with your changes are all failing with a syntax error:

```
script/functions.sh: line 90: syntax error near unexpected token `)'
```

Any ideas how your change could have caused that?

Comment (JonRowe) [inline travis/script/functions.sh]: Missing a "

Comment (bf4) [inline travis/script/functions.sh]: oy.

Comment (bf4) [inline travis/script/functions.sh]: Thanks @JonRowe That is deeply embarrassing.  Reminds me of the calculus exam I near got 100% on, but, on one of the questions, after solving some integrals, summed 1 and -1 to get 2, or something ridiculous like that.

Comment (JonRowe) [inline travis/script/functions.sh]: It's fine we all do it especially when we're not running the thing we're changing directly. The amount of times I commit 1.9 hash syntax into our specs then wonder why the build breaks ;)

**Comment by myronmarston** [comment]
OK, force pushed updates to all the PRs for all repos.

BTW, thanks for looking into this.  I had noticed the jruby warnings but wasn't sure how to fix them!

**Comment by bf4** [comment]
@myronmarston I had thought Travis was fixing the JRuby-head build cext.enabled warnings, but turns out that was something else.  One of us will have to check if there's an open issue for this.  cc @headius @enebo fwiw

**Comment by myronmarston** [comment]
Thanks, @bf4.  The rspec-mocks build passes on JRuby head now!

https://travis-ci.org/rspec/rspec-mocks/jobs/50015531

**Pull request merged**